---
title: "GENOMITexplorer"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

**GENOMITexplorer** is an open access online resource created as part of the study "Diagnosing pediatric mitochondrial disease: lessons from 2,000 exomes".

The resource contains:

**1) Information on the [variant-level](https://prokischlab.github.io/GENOMITexplorer/#Scripts_patient_variants_study_variants.html) for all “pathogenic” and “likely pathogenic” variants reported in the study.**

Variants are listed with their respective predicted function, heteroplasmy level (for mitochondrial DNA variants), allele frequency (according to gnomAD), CADD score, SIFT score, ACMG classification, functional evidence, and phenotype SS score.

**2) Information on HPO-gene associations, on the [patient-level](https://prokischlab.github.io/GENOMITexplorer/#Scripts_patient_HPO_study_plus_registry_and_literature_HPO.html) and [gene-level](https://prokischlab.github.io/GENOMITexplorer/#Scripts_patient_HPO_study_plus_registry_and_literature_HPO.html).**

HPO-gene association are provided for all patients included in the study in addition to pediatric molecularly diagnosed patients from mitochondrial disease registries (mitoNET and Besta) and from literature reports.

The most discriminating phenotypes by molecular diagnosis can be explored for all genes with ≥5 reported patients carrying “pathogenic” and “likely pathogenic” variants.

**All clinicians and researchers contributing to the data included in this resource are acknowledged [here](https://www.cmm.in.tum.de/project/exomes1000/html/#Scripts_contributors_contributors.html).**