<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="S.L.Stenton" />

<meta name="date" content="2021-09-03" />

<title>Genetic diagnoses</title>

<script src="libR/header-attrs-2.7/header-attrs.js"></script>
<script src="libR/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libR/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libR/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libR/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libR/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="libR/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="libR/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="libR/tocify-1.9.1/jquery.tocify.js"></script>
<script src="libR/navigation-1.1/tabsets.js"></script>
<script src="libR/navigation-1.1/codefolding.js"></script>
<link href="libR/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="libR/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="lib/add_content_table.css" type="text/css" />
<link rel="stylesheet" href="lib/leo_style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>



<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-sm-12 col-md-4 col-lg-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-sm-12 col-md-8 col-lg-9">




<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Genetic diagnoses</h1>
<h4 class="author">S.L.Stenton</h4>
<h4 class="date">2021-09-03</h4>

</div>


<p><strong>Note:</strong> all figures created as part of the study “Diagnosing pediatric mitochondrial disease by exome sequencing: lessons from 2,000 suspected cases”</p>
<p>by Stenton et al., (in submission) led by Dr. Holger Prokisch at the Technical University of Munich.</p>
<p><strong>Each of the figures on this page depicts the “pathogenic” (P) and “likely pathogenic” (LP) disease-causing variants</strong></p>
<p><strong>detected in the study.</strong></p>
<div id="wes-diagnostic-rate-stratified-by-molecular-cause" class="section level2">
<h2>WES diagnostic rate stratified by molecular cause</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-2-1.png" width="672" /></p>
<p>Number of solved and unsolved cases stratified by underlying molecular cause.</p>
<p><em><strong>Abbreviations:</strong> MD, mitochondrial disease; MD-phenocopy, mitochondrial disease phenocopy.</em></p>
</div>
<div id="inheritance-pattern" class="section level2">
<h2>Inheritance pattern</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-3-1.png" width="672" /></p>
<p>Proportion of molecular diagnoses by mode of inheritance in the MD and MD-phenocopy patients.</p>
</div>
<div id="frequent-genetic-diagnoses" class="section level2">
<h2>Frequent genetic diagnoses</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
<p>Genes in which P/LP variants were detected in ≥5 cases displayed with their frequency.</p>
<p>The most frequent genes with disease-causing variants in the MD-phenocopy patients are highlighted in light blue.</p>
</div>
<div id="number-of-cases-per-genetic-diagnosis" class="section level2">
<h2>Number of cases per genetic diagnosis</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-5-1.png" width="672" /></p>
<p>Number of cases per gene with P/LP variants.</p>
</div>
<div id="breakdown-of-md-phenocopy-genetic-diagnoses" class="section level2">
<h2>Breakdown of MD-phenocopy genetic diagnoses</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-6-1.png" width="672" /></p>
<p>Number of MD-phenocopy cases by disease gene category.</p>
<p><em><strong>Abbreviations:</strong> NDD, neurodevelopmental disease; NMD, neuromuscular disease; IEM, inborn errors of metabolism; ND, neurodegeneration; CM, cardiomyopathy.</em></p>
</div>
<div id="variant-function" class="section level2">
<h2>Variant function</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-7-1.png" width="672" /></p>
<p>Proportion of variants of each functional class in all solved cases collectively.</p>
<p><em><strong>Abbreviations:</strong> CNV, copy number variant; mtDNA, mitochondrial DNA.</em></p>
</div>
<div id="variant-function-stratified-by-molecular-diagnosis" class="section level2">
<h2>Variant function stratified by molecular diagnosis</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-8-1.png" width="672" /></p>
<p>Proportion of variants of each functional class in the MD patients and MD-phenocopy patients.</p>
</div>
<div id="number-of-cases-per-missense-variant" class="section level2">
<h2>Number of cases per missense variant</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-9-1.png" width="672" /></p>
</div>
<div id="validation-assay-for-missense-variants" class="section level2">
<h2>Validation assay for missense variants</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-10-1.png" width="672" /></p>
<p>Functional validation assay leading to designation of the missense variant as P/LP.</p>
<p><em><strong>Abbreviations:</strong> RCC, mitochondrial respiratory chain complex; OCR, oxygen consumption rate.</em></p>
</div>
<div id="variant-allele-frequency" class="section level2">
<h2>Variant allele frequency</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-11-1.png" width="672" /></p>
<p>Variants ranked by gnomAD allele frequency. Routinely applied allele frequency thresholds of 1% and 0.1% are indicated. Two frequent variants in <em>DARS2</em> were confirmed to be pathogenic as a rare combination in <em>cis</em>, confirmed by RNA seq and proteomic analyses as reported by <a href="https://www.medrxiv.org/content/10.1101/2021.03.09.21253187v1" target="_blank">Kopajtich et al., 2021</a>.</p>
</div>
<div id="variant-cadd-score" class="section level2">
<h2>Variant CADD score</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-12-1.png" width="672" /></p>
<p>Variants ranked by CADD score. CADD scores ≥20 and ≥40 are indicated. Variants above these thresholds are within the 1% and 0.1% most deleterious substitutions, respectively.</p>
</div>
<div id="variant-sift-score" class="section level2">
<h2>Variant SIFT score</h2>
<p><img src="Scripts_WES_study_genetic_diagnoses_files/figure-html/unnamed-chunk-13-1.png" width="672" /></p>
<p>Variants ranked by SIFT score. respectively. SIFT scores &lt;0.05 are predicted to be deleterious.</p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
